### PR TITLE
remove first rows fallback variable

### PR DESCRIPTION
**Pull request by JatinKumar001**
Remove the FIRST_ROWS_FALLBACK_MAX_DATASET_SIZE from the code.

### CODE REVIEW

**Review by albertvillanova — CHANGES_REQUESTED** [review]
Thanks for the fix @JatinKumar001.

We are working on a related issue (thanks @severo).
- #773

Once this PR is merged, could you please merge main branch into your branch?
```
git checkout remove-FIRST_ROWS_FALLBACK_MAX_DATASET_SIZE
git fetch upstream main
git merge upstream/main
git push -u origin remove-FIRST_ROWS_FALLBACK_MAX_DATASET_SIZE
```

**Review by severo — CHANGES_REQUESTED** [review]
Thanks a lot!

One small additional change is needed, and we're good!

**Comment by severo** [comment]
Note that the CI is failing due to a temporary issue with the GitHub Actions (Azure repository of Ubuntu packages). It's not linked to the PR, and I'll relaunch the actions later, when the outage is over.

File: chart/templates/worker/first-rows/_container.tpl [hunk 1]
```diff
@@ -24,8 +24,6 @@
     # value: {{ .Values.queue.maxJobsPerNamespace | quote }}
     # overridden
     value: {{ .Values.firstRows.queue.maxJobsPerNamespace | quote }}
-  - name: FIRST_ROWS_FALLBACK_MAX_DATASET_SIZE
-    value: {{ .Values.firstRows.fallbackMaxDatasetSize | quote }}
```
[the remark this answers is not in the document]
Comment (JatinKumar001) [reply chart/templates/worker/first-rows/_container.tpl:28]: Ok sure

**Comment by JatinKumar001** [comment]
Hey, can you tell me what I have to do to correct these errors?

**Comment by severo** [comment]
Thanks.

For the CI errors, we have to wait until Azure fixes its repository, and then relaunch the github actions.

**Comment by albertvillanova** [comment]
We have to wait until Canonical fixes their error on their servers:
- https://github.com/actions/runner-images/issues/7048

**Comment by severo** [comment]
To finish this PR, you should:
- merge main into your branch
  ```
    git checkout remove-FIRST_ROWS_FALLBACK_MAX_DATASET_SIZE
    git fetch upstream main
    git merge upstream/main
    git push -u origin remove-FIRST_ROWS_FALLBACK_MAX_DATASET_SIZE
    ```
- remove the two lines in `chart/values.yaml` at https://github.com/huggingface/datasets-server/blob/3bb80b725c361bb5072e14ffc82ab4838ad12405/chart/values.yaml#L285-L286

**Comment by severo** [comment]
It seems like you still have not merged the main branch of this repository into your branch. Do you want us to help with that?

**Comment by JatinKumar001** [comment]
I done it and also create a new pull request 
#781

**Comment by severo** [comment]
It seems like the branch you merged into your own branch is not the current main branch.

**Review by albertvillanova — APPROVED** [review]
@JatinKumar001 please note that you don't have to open a new pull request.

You only need to push to this PR the requested changes.

I think we have fixed the problems and this PR is already OK for merging. Do you agree @severo?
EDIT: we need to fix the CI errors first.

**Review by albertvillanova — CHANGES_REQUESTED** [review]
CI errors should be fixed first.

**Review by albertvillanova — APPROVED** [review]

**Review by severo — APPROVED** [review]
Thanks @JatinKumar001 for the contribution!

And thanks @albertvillanova to make it mergeable and for finding the bug (https://github.com/huggingface/datasets-server/pull/771/commits/712e87189b34f591fea5544f3b3e711d2fbfb95b)!

**Pull request merged**